deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -509,23 +695,27 @@ services:
       - storage-users
       - server
     environment:
-      OCIS_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
-      OCIS_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
-      OCIS_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+      STORAGE_USERS_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      STORAGE_USERS_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      STORAGE_USERS_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
 
       STORAGE_USERS_DATAPROVIDER_INSECURE: "${INSECURE:-false}"
 
       STORAGE_USERS_DRIVER: ocis
 
       STORAGE_USERS_GRPC_ADDR: 0.0.0.0:9157
-      STORAGE_USERS_ENDPOINT: storage-users:9157
-
       STORAGE_USERS_HTTP_ADDR: 0.0.0.0:9158
+
       STORAGE_USERS_DATA_SERVER_URL: http://storage-users:9158/data
 
-      STORAGE_TRANSFER_SECRET: ${STORAGE_TRANSFER_SECRET:-replace-me-with-a-transfer-secret}
-      STORAGE_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
-      REVA_GATEWAY: storage-gateway:9142
+      STORAGE_USERS_PERMISSION_ENDPOINT: settings:9191
+      STORAGE_USERS_EVENTS_ENDPOINT: nats:9233
+
+      REVA_GATEWAY: gateway:9142
+
+      STORAGE_USERS_JWT_SECRET: ${OCIS_JWT_SECRET}
+      STORAGE_USERS_TRANSFER_SECRET: ${STORAGE_TRANSFER_SECRET}
```

Comment:
according to https://owncloud.dev/extensions/storage-users/configuration/, `STORAGE_USERS_TRANSFER_SECRET` does not exist